examples/kyb-app/src/components/organisms/UIRenderer/elements/Task/Task.tsx

```diff
@@ -0,0 +1,34 @@
+import { baseElements } from '@app/components/organisms/UIRenderer/base-elements';
+import { BlocksComponent } from '@ballerine/blocks';
+import { Card } from '@ballerine/ui';
+import { CSSProperties } from 'react';
+
+export interface TaskOptions {
+  styles?: CSSProperties;
+  className?: string;
+  variant?: 'wrapper' | 'card';
+}
+
+export interface TaskProps {
+  options?: TaskOptions;
+  childrens: any;
+}
+
+export const Task = ({ options = {}, childrens, ...rest }: TaskProps) => {
+  const { styles, className, variant } = options;
+
+  const Component = variant === 'wrapper' ? Card : 'div';
+
+  return (
+    <Component className={className} style={styles} data-test-id="task">
```

Comment:
Avoid test ids. Tested components should be selectable via selectors or the test potentially needs a revisit.

Reply:
This is leftover :D